.gitpod.yml

```diff
@@ -1,23 +1,43 @@
-image:
-  file: .gitpod.Dockerfile
+tasks:
+  - name: Setup Sail environment and Composer dependencies
+    openMode: split-right
+    # Fix because of https://github.com/gitpod-io/gitpod/issues/16614
+    before: sudo curl -o /usr/local/bin/docker-compose -fsSL https://github.com/docker/compose/releases/download/v2.16.0/docker-compose-linux-$(uname -m)
+    init: |
+      cp .env.example .env &&
+      sed -i "s#APP_URL=http://localhost#APP_URL=$(gp url 80)#g" .env
+      sed -i "s#GITPOD_VITE_URL=#GITPOD_VITE_URL=$(gp url 5173)#g" .env
+      composer install --ignore-platform-reqs
+      ./vendor/bin/sail up -d
+      ./vendor/bin/sail php artisan key:generate
+      ./vendor/bin/sail php artisan storage:link
+      cat .gitpod.welcome-message
+      gp sync-done sail-is-ready
+
+  - name: Install Node dependencies and run Vite
+    openMode: split-right
```

Comment:
I think, we should not split this part with the Gitpod message. As it is getting not readable


![image](https://user-images.githubusercontent.com/55068936/229435914-c965f2bd-ec19-4400-b19f-89e30c5178cf.png)


```suggestion
```

Reply:
I'll do that rigth away and then you see it with your eyes and decide :)